### DISCUSSION TITLE
url to license server

### DISCUSSION THREAD

**Discussion by alpacapps** [spatie/ray#739] [General]
I have a client where i'm working on location, that have a very strict policy about outgoing traffic. Every url needs to be authorized by the firewall. 
When using ray, it tries to fetch the license but i am greeted with a server error because it isn't allowed to make an external call. What is the root url for fetching the licenses? That way i can ask to add it to the excluded domains.

**Comment by freekmurze** [comment]
You should ask to whitelist URLs that start with these:

- https://spatie.be/api/license
- https://spatie.be/is-valid-license
